xmhf/src/xmhf-core/include/arch/x86/_processor.h

```diff
@@ -157,6 +157,8 @@
 //SDM Vol 2B. 4-427)
 struct regs
 {
+  u32 flags;
```

Comment:
As mentioned in email, it is easy to make mistakes when changing `struct regs`. For example, is `xmhf/src/xmhf-core/xmhf-runtime/xmhf-xcphandler/arch/x86_64/xcph-stubs.S` also using this struct?
It is better to use
```c
struct some_other_name {
	struct regs gprs;
	u32 eflags;
	u32 eip;
	u16 cs;	// if needed
	u16 ds;	// if needed
	u16 ss;	// if needed
	...
}
```
And
```c
static inline ulong_t CPU_Reg_Read(struct regs* r, enum CPU_Reg_Sel sel)
{
    switch (sel)
    {
        case CPU_REG_AX:
            return r->gpr.rax;
        case CPU_REG_BX:
            return r->gpr.rbx;
        ...
        case CPU_REG_FLAGS:
            return r->eflags;
        case CPU_REG_IP:
            return r->eip;
        ...
```